spec/fixtures/state_file/fed_return_xmls/2023/nj/zeus_one_dep.xml

```diff
@@ -147,6 +147,9 @@
       <StudentLoanInterestDedAmt>1300</StudentLoanInterestDedAmt>
       <TotalAdjustmentsAmt>1500</TotalAdjustmentsAmt>
     </IRS1040Schedule1>
+    <IRS1040Schedule3>
```

Comment:
[DUST] How was this XML sourced? Did we copy it from another state's XML example?

Reply:
I inferred it from the directfile import. Feels sketchy but I was not sure how else to accomplish this

Reply:
Gotcha, seems fine.

Reply:
Did you look in some of the other DF state imports?

Reply:
Yes, this was not present in any others

Reply:
This field was not in another DF import. I manually added it here because we needed it to exist to test this PR, and it could be inferred from the df import code.